### PR TITLE
Add WSHP

### PR DESCRIPTION
**Pull request by briancjohns**
Added a new cooling only, zone temp controlled WSHP that uses condenser water to carry heat away from IDF rooms.

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml [hunk 1]
```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
...
   - POTHWSC
 
+WSHP_FDPM_CWDT_SFVSC_CDCPC_CAM_ZTC_RTM_CWDT_CDWISOVM_STM:
+  guid: "e1b4a911-00a8-42da-abaf-8eff6987ad58"
+  description: "Cooling only water source heat pump using condensor water to carry heat away from the zone.  One filter, condensor water pump control, one compressor, condensate alarm"
+  is_canonical: true
+  implements:
+  - WSHP
+  - FDPM
+  - CWDT
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:37]: You use `CWDT` twice, please fix.

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml [hunk 2]
```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
...
+  - SFVSC
+  - CDCPC
+  - CAM
+  - ZTC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:41]: How does this unit control to zone air temperature? I would expect it to control to a supply or return water temperature setpoint.

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml [hunk 3]
```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
...
+  - RTM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:42]: Return air or return water?

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml [hunk 4]
```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
...
+  - CWDT
+  - CDWISOVM
+  - STM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:45]: Supply Air or supply water?

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml [hunk 5]
```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
...
 
+WSHP_FDPM_CWDT_SFVSC_CDCPC_CAM_ZTC_RTM_CWDT_CDWISOVM_STM:
+  guid: "e1b4a911-00a8-42da-abaf-8eff6987ad58"
+  description: "Cooling only water source heat pump using condensor water to carry heat away from the zone.  One filter, condensor water pump control, one compressor, condensate alarm"
+  is_canonical: true
+  implements:
+  - WSHP
+  - FDPM
+  - CWDT
+  - SFVSC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:38]: Typically we don't see this on water-side equipment, what does this fan speed represent?

**Review by cstirdivant — COMMENTED** [review]
@briancjohns please fix validator errors.

I'm seeing a lot of air-side points on this equipment, which is not consistent with this being a water-side device

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml (hunk 1 above)
Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:37]: I sent an invite for a discussion tomorrow.  This WSHP has return air drawn through a coil.  In the coil is condenser water from the plant.  The condenser water is cooled via DX with a compressor using refrigerant.  Heat is taken out of the space and put into the condenser water loop.

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml (hunk 2 above)
Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:41]: It controls to zone air, regulating the compressor to attain zone temp setpoint.  The compressor uses refrigerant to cool a condenser water loop within the unit.  The cooled condenser water flows through an air/condenser water coil.

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml (hunk 4 above)
Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:45]: supply air

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml (hunk 3 above)
Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:42]: return air

File: ontology/yaml/resources/HVAC/entity_types/WSHP.yaml (hunk 5 above)
Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/WSHP.yaml:38]: It is the supply air fan pulling air through the WSHP.

**Pull request closed without merging**